packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -236,7 +245,13 @@ export const prepareStatusManager = zone => {
        */
       forwarded(txHash, address, amount) {
         // TODO: store txHash -> evidence for txs pending settlement?
-        console.log('TODO: vstorage update', { txHash, address, amount });
+        if (txHash) {
+          recordStatus(txHash, 'FORWARDED');
+        } else {
+          console.log(
+            `Forwarded minted amount ${amount} from account ${address} before it was observed.`,
```

Comment:
```suggestion
            `⚠️ Forwarded minted amount ${amount} from account ${address} before it was observed.`,
```

cf [Logging](https://github.com/Agoric/agoric-sdk/wiki/Logging)

and
> C18 - Contract MUST implement logic to handle settlements without corresponding advances
>   - MUST log and alert these incidents

Reply:
Done